schemas/dependencies/jenkins-instance-1.yml

```diff
@@ -50,6 +50,39 @@ properties:
       required:
       - name
       - keep_hours
+  workerFleets:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
```

Comment:
I would add a description field here stating how this options come from https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/docs/